iotdb/src/main/java/org/apache/iotdb/db/utils/LoadDataUtils.java

```diff
@@ -133,7 +133,7 @@ private void loadOneRecordLine(String line) {
     try {
       nsPath = mmanager.getFileNameByPath(record.deviceId);
     } catch (PathErrorException e) {
-      LOG.error("given seriesPath not found.{}", e.getMessage());
+      logger.error("given seriesPath not found, given deviceId:{} ", record.deviceId, e);
```

Comment:
only one `{}` ??
same as below.

Reply:
The `{}` within the text will be filled by the second parameter(`deivceId`). The last `e`(Exception) needn't  `{}`, and all message and stack trace of `e` will be print out。